vllm/utils.py

```diff
@@ -167,6 +167,13 @@ def random_uuid() -> str:
     return str(uuid.uuid4().hex)
 
 
+def get_vllm_version() -> str:
+    # Avoid circular import
+    import vllm
```

Comment:
If you define the VERSION constant here, then set `vllm.__version__ = vllm.utils.get_vllm_version()`, you can avoid this ugly import